app/main.js

```diff
@@ -26,49 +57,57 @@ function createWindow () {
   })
 };
 
+function handleDaemonSubprocessExited() {
+  console.log('The daemon has exited.');
+  daemonSubprocess = null;
+  if (!daemonSubprocessKillRequested) {
+    // We didn't stop down the daemon subprocess on purpose, so display a
+    // warning and schedule a quit.
+    //
+    // TODO: maybe it would be better to restart the daemon?
+    console.log('Did not display, so scheduling quit');
```

Comment:
No idea where that came from. Maybe just a copy-and-paste error. I meant for it to say something like "did not intentionally quit daemon, so scheduling quit."